python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/vxlan_interface.schema.yml

```
@@ -100,6 +100,23 @@ keys:
                       items:
                         type: str
                         description: Remote VTEP IP Address.
+              vlan_range:
+                type: dict
+                description: |-
+                  Set VNI on range of vlans.
+                  If vni is set on the vlan in `vxlan_interface.vxlan1.vxlan.vlans`, it should not overlap with `vxlan_interface.vxlan1.vxlan.vlans`.
```

`vxlan_interface.vxlan1.vxlan.vlans` is duplicated
```suggestion
                  If vni is set on the vlan in `vxlan_interface.vxlan1.vxlan.vlans`, it should not overlap with `vxlan_interface.vxlan1.vxlan.vlan_range`.
```